lib/assignDisabledRanges.mjs

```diff
@@ -147,7 +147,7 @@ export default function assignDisabledRanges(root, result) {
 		}
 
 		if (isAtRule(node)) {
-			let offset = node.name.length + 1;
+			let offset = node.name.length + 1; // `@` + name
```

Comment:
nit
style

In those cases I prefer putting the documentation in the code itself.
e.g. `'@'.length + node.name.length`

Reply:
But that is a massive increase in computational effort.

I now that the absolute effort is still small, but still feels wrong to waste CPU cycles when a small code comment can serve the same purpose.